site.someones.Lonewolf.appdata.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>site.someones.Lonewolf</id>
+
+  <name>Lonewolf</name>
+  <summary>Organize and track your tasks with ease and flexibility.</summary>
+  <developer_name>Mario Aichinger</developer_name>
+
+  <metadata_license>FSFAP</metadata_license>
+  <project_license>GPL-3.0-only</project_license>
+
+  <url type="homepage">https://github.com/aichingm/Lonewolf</url>
+  <url type="vcs-browser">https://github.com/aichingm/Lonewolf</url>
+  <url type="bugtracker">https://github.com/aichingm/Lonewolf/issues</url>
+
+  <supports>
+    <internet>offline-only</internet>
+    <control>pointing</control>
+    <control>keyboard</control>
+    <display_length compare="ge">small</display_length>
+    <display_length compare="le">xlarge</display_length>
+  </supports>
+
+  <description>
+    <p>
+      Lonewolf is a productivity application based on the principles of Kanban.
+    </p>
+    <ul>
+      <li>Manages lists</li>
+      <li>Create cards</li>
+      <li>Organize cards with lables</li>
+      <li>Set due dates for yourself</li>
+      <li>Add attchments</li>
+      <li>Write descripotions of your tasks in markdown</li>
+      <li>Split up larger tasks with check lists</li>
+      <li>Keep your thoughts by adding comments for your future self</li>
+    </ul>
+  </description>
+
+  <launchable type="desktop-id">site.someones.Lonewolf.desktop</launchable>
+
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/flathub/site.someones.Lonewolf/master/assets/screenshot_light_main.png</image>
```

Comment:
this URL doesn't exist. That mean the validation WILL fail.

Reply:
I see. I added the screen shots in anticipation of the repository being created. I will just comment them out.

Reply:
that will fail too. at least ONE screenshot is required.

Reply:
Oh. Since the xml should be upstreamed I can also add the screenshots upstream.

Reply:
definitely. 

and the docker thing-a-mabob.

Reply:
Would it be a problem to keep it in this repo, since it is used by the Makefile to run `flatpak-builder` locally.?

Reply:
why do you need Docker?

Reply:
I don't actually need docker podman is fine too. I like to compartmentalize build environments so that I can easily run them on different machines without the need to replicate the build environment on all of them and keep them in sync.